internal/pkg/describe/uri.go

```diff
@@ -265,35 +269,35 @@ func (d *albDescriber) uri() (albURI, error) {
 
 	lbDescr, err := d.initLBDescriber(d.env)
 	if err != nil {
-		return albURI{}, nil
+		return accessURI{}, nil
 	}
 	dnsNames, err := lbDescr.ListenerRuleHostHeaders(ruleARN)
 	if err != nil {
-		return albURI{}, fmt.Errorf("get host headers for listener rule %s: %w", ruleARN, err)
+		return accessURI{}, fmt.Errorf("get host headers for listener rule %s: %w", ruleARN, err)
 	}
 	if len(dnsNames) == 0 {
 		return d.envDNSName(path)
 	}
-	return albURI{
+	return accessURI{
 		HTTPS:    httpsEnabled,
 		DNSNames: dnsNames,
 		Path:     path,
 	}, nil
 }
 
-func (d *albDescriber) bestEffortRemoveEnvDNSName(albURI albURI) albURI {
+func (d *uriDescriber) bestEffortRemoveEnvDNSName(accessURI accessURI) accessURI {
```

Comment:
```suggestion
func (d *uriDescriber) bestEffortRemoveALBDNSName(accessURI accessURI) accessURI {
```

small nit, maybe we should change this func name along with the struct key change